client/js/table-view.js

```diff
@@ -74,10 +76,50 @@ class TableView{
         this.formulaBarEl.addEventListener('keyup', this.handleFormulaBarChange.bind(this));
     }
 
+    renderSum(collie, total) {
+        const whichColumn = document.querySelector(`#sum-${collie}`);
+        whichColumn.textContent = total;
+    }
+    
+    sumOf(arr) {
+        return arr.reduce(((total, x) => total+=x),0);
+    }
+
+    fishForNumbers(arr) {
+        return arr.map(x => Number(x))
+                  .filter(x => !(isNaN(x)));
+    }
+
+    getAllDataInColumn(col, inc, memo) {
```

Comment:
This memo thing is an interesting idea but I think you should get rid of it...  

It breaks "encapsulation" to push in a data structure from outside of the function.  It's much better if `getAllDataInColumn` is a self-contained entity, a "black box", that will just get all the data for a given column index.  

What would be even better actually is moving this entire function inside of the model.  Getting data is what the model is for right?

There's absolutely no need for this to be a recursive function either. It's not a good use case for it. We will encounter good use cases for it in about a week, so hold your horse(s) for that and make this a simple loop please.